src/chigame/api/tests/test_api.py

```diff
@@ -46,7 +46,7 @@ def test_update_game(self):
 
         # Update the game object
         url = reverse("api-game-detail", args=[game.id])
-        updated_data = {"name": "The Witcher 3: Wild Hunt", "max_players": 5}
+        updated_data = {"name": "The Witcher 3: Wild Hunt", "max_players": 10}
```

Comment:
Why was this change made?

Reply:
The test would break nondeterministically if minimum players happened to be above 5.